wave_benchmark/include/wave/benchmark/pose_measurement.hpp

```diff
@@ -16,11 +16,11 @@ namespace wave {
  *  @{ */
 
 struct BenchmarkPose {
-    Rotation rotation;
+    RotationMd rotation;
     Vec3 translation;
 
-    BenchmarkPose() : rotation(Rotation()), translation{0.0, 0.0, 0.0} {}
-    BenchmarkPose(Rotation rot, Vec3 trans)
+    BenchmarkPose() : rotation(RotationMd()), translation{0.0, 0.0, 0.0} {}
```

Comment:
Does this change behaviour? `RotationMd` is initialized to garbage, but old `Rotation` was initialized to identity IIRC